public/js/question.js

```diff
@@ -90,8 +90,18 @@ var sendAnswerRequest = function(ans) {
             $('#modalAlert').modal('open');
         },
         error: function(data) {
-            $('#hint').removeClass('hidden');
-            swal('Incorrect', 'Sorry, that\'s the wrong answer', 'error');
+            if (data['status'] === 401) {
+                window.location.href = '/';
```

Comment:
There seem to be a lot of empty spaces after this line, please take a look at that

Reply:
Not sure where the spaces are, Just checked